certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -2368,25 +2301,25 @@
                 },
                 "taskid": {
                   "type": "string",
-                  "description": "taskid",
+                  "description": "card id",
                   "title": "Card ID",
                   "x-ms-visibility": "important"
                 },
                 "history": {
                   "type": "string",
-                  "description": "Set to “yes” if you want to get history for the task",
+                  "description": "set to \u201cyes\u201d if you want to get card history",
```

Comment:
@kostadin-kanbanize unicode symbols are not allowed. Please remove them. You can use apostrophe sign instead, e.g. `"Set to 'yes' if..."`

Reply:
Unicode characters removed